__tests__/index.test.ts

```diff
@@ -243,4 +243,29 @@ describe('E2E', () => {
       (<any>expect(result)).toMatchSpecificSnapshot(join(folderPath, 'remove-unused-components-snapshot.js'));
     });
   });
+
+  describe('webpack-bundle test', () => {
```

Comment:
Could we maybe put the smoke test into a separate file in order not to mix with functional tests?

Reply:
I think yes, this make sense!